docs/lofi-hifi-backends.md

```diff
@@ -0,0 +1,212 @@
+# Introduce Commands that Run Jobs with Different Fidelity levels for Key ilab functions
+
+This document describes adding different data generation, mixing, and training backends to ilab to enable higher fidelity training using the backend code.
+
+Currently all training is done via qlora or the like. Adding the following commands will enable higher fidelity training and introduce commands such as data mixing.
+
```

Comment:
A major part of this proposal is integrating phased training, though it doesn't really explain anywhere what that means. It would be helpful in the introduction to recap the current state of training support and then introduce the major new training capabilities proposed here. You can refer back to the original Lab paper for a more detailed description.

https://arxiv.org/html/2403.01081v3

The same issue applies to data generation. The doc doesn't really explain why `lofi/hifi` commands are necessary. An overview of the current state, plus an overview of what's proposed to be added, along with why it needs to be a separate command, would be helpful.

If you don't want to get into all of that for the data generation side, you could focus this proposal on training only.